_releases/2022-01-06-1.3.0-released.md

```diff
@@ -22,6 +22,32 @@ In this release we included [XXX PRs since the 1.2.2 release](https://github.com
 
 ## Multi-Assign
 
+Multiple assignments got improved in various ways. First, it's possible to take a "splat of elements" in a multi-assignment:
+
+<div class="code_section">{% highlight crystal %}
+# Splat in multi-assign with array
+first, *rest, last = [1, 2, 3, 4, 5]
+first # => 1
+rest # => [2, 3, 4]
+last # => 5
+{% endhighlight %}</div>
+
+<div class="code_section">{% highlight crystal %}
+# Splat in multi-assign with tuple
+*rest, last = {"This", 15, 4, "tuple", true}
+rest # => {"This", 15, 4, "tuple"}
+last # => true
+{% endhighlight %}</div>
+
+Second, there's an optional preview feature to detect unbalanced multi-assignments. It can be enabled with the compiler flag `-Dstrict_multi_assign`.
+
+<div class="code_section">{% highlight crystal %}
+a, b, c = {1, 2} # Error: index out of bounds for Tuple(Int32, Int32) (2 not in -2..1)
+a, b = {1, 2, 3} # Error: cannot assign Tuple(Int32, Int32, Int32) to 2 targets
+{% endhighlight %}</div>
+
+⚠️ This second improvement is a breaking change; therefore it's not enabled by default. We encourage using it to detect possible errors in your code. This setting will likely be the default in 2.0. An additional restriction guarded by that flag is that if the right-hand side has a unique element, it must be of type `Indexable` or `Enumerable`.
+
```

Comment:
```suggestion

If you actually relied on ignoring the "overflow" items, this syntax will achieve the old behavior:

<div class="code_section">{% highlight crystal %}
a, b, *_ = [1, 2, 3]
# Same as the old: a, b = [1, 2, 3]
{% endhighlight %}</div>
```

Reply:
Good addition. But I think this should be moved up at the end of the first part. And instead of a vague "this syntax", I'd explicitly call the underscore splat target.